src/test/java/uk/gov/homeoffice/digital/sas/balancecalculator/kafka/consumer/TimeEntryConsumerTest.java

```diff
@@ -44,7 +49,13 @@
 class TimeEntryConsumerTest {
 
   @Captor
-  private ArgumentCaptor<TimeEntry> timeEntryCaptor;
+  private ArgumentCaptor<TimeEntry> timeEntryCaptor1;
```

Comment:
Does this variable name need to have `1` in it?

Reply:
Good catch, i was meant to remove that